src/functionalTest/resources/scenarios/RWA-318-Allocate-a-case-to-caseworker-midevent.json

```diff
@@ -1,6 +1,6 @@
 {
   "description": "RWA-318 Allocate a case to caseworker midEvent",
-  "disabled": "false",
+  "disabled": "true",
```

Comment:
RWA tests needs to work before merge to the master, we should not disable them

Reply:
i reset this test on master and it is not part of my PR anymore